index.html

```diff
@@ -175,7 +175,7 @@ <h2 id="docconformance">
         The following table provides normative per-element document-conformance
         requirements for the use of ARIA markup in HTML documents and describes
         the <a>implicit ARIA semantics</a> that apply to
-        <a data-cite="html/dom.html#elements">HTML elements</a> as defined in [[html-aam-1.0|HTML AAM]].
+        [=HTML elements=] as defined in [[html-aam-1.0|HTML AAM]].
```

Comment:
similar with previous comment.  not where i'd want this HTML elements link to go

Reply:
As above. I also think we should fix the wording of this to:

> The following table provides normative per-element document-conformance  requirements for the use of ARIA markup in HTML documents and describes the <a>implicit ARIA semantics</a>, as defined in [[html-aam-1.0|HTML AAM]], that apply to [=HTML elements=] . 

Otherwise it reads like HTML AAM defines the HTML elements, which is not right.

WDYT?

Reply:
well html aam defines the mappings, which is right.  but regardless of that, per my comment on the previous instance of this, we can make this change if we get the semantics link in place.